tests/integration/converters/test_png.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+import numpy as np
+import pytest
+from PIL import Image, ImageChops
+
+import tiledb
+from tests import assert_image_similarity, get_path, get_schema
+from tiledb.bioimg.converters import DATASET_TYPE, FMT_VERSION
+from tiledb.bioimg.converters.png import PNGConverter
+from tiledb.bioimg.helpers import open_bioimg
+from tiledb.bioimg.openslide import TileDBOpenSlide
+from tiledb.cc import WebpInputFormat
+
+
+def test_png_converter(tmp_path):
+    input_path = str(get_path("A01_s1--cell_outlines.png"))
+    output_path = str(tmp_path)
+
+    PNGConverter.to_tiledb(input_path, output_path)
+
+    with TileDBOpenSlide(output_path) as t:
+        assert len(tiledb.Group(output_path)) == t.level_count == 1
+        schemas = get_schema(1080, 1080)
+        assert t.dimensions == schemas.shape[:-3:-1]
+        for i in range(t.level_count):
+            assert t.level_dimensions[i] == schemas.shape[:-3:-1]
+            with open_bioimg(str(tmp_path / f"l_{i}.tdb")) as A:
+                assert A.schema == schemas
+
+        region = t.read_region(level=0, location=(100, 100), size=(300, 400))
+        assert isinstance(region, np.ndarray)
+        assert region.ndim == 3
+        assert region.dtype == np.uint8
+        img = Image.fromarray(region)
+        assert img.size == (300, 400)
+
+        for level in range(t.level_count):
+            region_data = t.read_region((0, 0), level, t.level_dimensions[level])
+            level_data = t.read_level(level)
+            np.testing.assert_array_equal(region_data, level_data)
+
+
+@pytest.mark.parametrize("filename", ["A01_s1--cell_outlines.png"])
+def test_png_converter_group_metadata(tmp_path, filename):
+    input_path = get_path(filename)
+    tiledb_path = tmp_path / "to_tiledb"
```

Comment:
```suggestion
    tiledb_path = str(tmp_path / "to_tiledb")
```

and then you won't need the two `str(tiledb_path)` below